habitat-baselines/habitat_baselines/config/experiments_hab3/pop_play_kinematic_oracle_humanoid_spot.yaml

```diff
@@ -60,6 +64,7 @@ habitat:
       - agent_0_is_holding
       - agent_0_ee_pos
       - agent_0_localization_sensor
+      - agent_0_other_agent_gps
       - agent_0_has_finished_oracle_nav
```

Comment:
the conclusion is not to use the depth image of Spot?

Reply:
We will use depth image, but also the other agent gps. I think we need both for social rearrangement right now.